docs/src/man/querying_frameworks.md

```diff
@@ -12,59 +12,111 @@ A query is started with the `@from` macro and consists of a series of query comm
+```jldoctest
+julia> using DataFrames, Query
+
+julia> df = DataFrame(name=["John", "Sally", "Roger"], age=[54., 34., 79.], children=[0, 2, 4])
+3×3 DataFrames.DataFrame
+│ Row │ name  │ age  │ children │
+├─────┼───────┼──────┼──────────┤
+│ 1   │ John  │ 54.0 │ 0        │
+│ 2   │ Sally │ 34.0 │ 2        │
+│ 3   │ Roger │ 79.0 │ 4        │
+
+julia> q1 = @from i in df begin
+            @where i.age > 40
+            @select {number_of_children=i.children, i.name}
+            @collect DataFrame
+       end
+2×2 DataFrames.DataFrame
+│ Row │ number_of_children │ name  │
+├─────┼────────────────────┼───────┤
+│ 1   │ 0                  │ John  │
+│ 2   │ 4                  │ Roger │
 
-q1 = @from i in df begin
-     @where i.age > 40
-     @select {number_of_children=i.children, i.name}
-     @collect DataFrame
-end
 ```
 
 The query starts with the `@from` macro. The first argument `i` is the name of the range variable that will be used to refer to an individual row in later query commands. The next argument `df` is the data source that one wants to query. The `@where` command in this query will filter the source data by applying the filter condition `i.age > 40`. This filters out any rows in which the `age` column is not larger than 40. The `@select` command then projects the columns of the source data onto a new column structure. The example here applies three specific modifications: 1) it only keeps a subset of the columns in the source `DataFrame`, i.e. the `age` column will not be part of the transformed data; 2) it changes the order of the two columns that are selected; and 3) it renames one of the columns that is selected from `children` to `number_of_children`. The example query uses the `{}` syntax to achieve this. A `{}` in a Query.jl expression instantiates a new [NamedTuple](https://github.com/blackrock/NamedTuples.jl), i.e. it is a shortcut for writing `@NT(number_of_children=>i.children, name=>i.name)`. The `@collect` statement determines the data structure that the query returns. In this example the results are returned as a `DataFrame`.
 
 A query without a `@collect` statement returns a standard julia iterator that can be used with any normal julia language construct that can deal with iterators. The following code returns a julia iterator for the query results:
 
-```@example 1
-q2 = @from i in df begin
-     @where i.age > 40
-     @select {number_of_children=i.children, i.name}
-end
-nothing # hide
+```jldoctest
+julia> using DataFrames, Query
+
+julia> df = DataFrame(name=["John", "Sally", "Roger"], age=[54., 34., 79.], children=[0, 2, 4])
+3×3 DataFrames.DataFrame
+│ Row │ name  │ age  │ children │
+├─────┼───────┼──────┼──────────┤
+│ 1   │ John  │ 54.0 │ 0        │
+│ 2   │ Sally │ 34.0 │ 2        │
+│ 3   │ Roger │ 79.0 │ 4        │
+
+julia> q2 = @from i in df begin
+            @where i.age > 40
+            @select {number_of_children=i.children, i.name}
+       end;
```

Comment:
Is it so ugly that we can't print it? That would be clearer, even if it's not really useful.